licenses.txt

```diff
@@ -1,62 +1,419 @@
 
-Licenses
-========
+Managed manually
+----------------
 
-Backbone.js
-===========
-http://backbonejs.org/
+Library: backbone.js
+Site http://backbonejs.org/
 License: MIT
 
-
-Underscore
-==========
-http://underscorejs.org/
+Library: backbone-localstorage
+Site: https://github.com/jeromegn/Backbone.localStorage
 License: MIT
 
-
-Backbone.localStorage
-=====================
-https://github.com/jeromegn/Backbone.localStorage
+Library: backbone.mousetrap
+Site: https://github.com/elasticsales/backbone.mousetrap
 License: MIT
 
+Library: bootstrap3
+Site: http://getbootstrap.com/
+License: Apache Software License, Version 2
 
-Bootstrap
-=========
-http://getbootstrap.com/
-License: Apache
-
+Library: bootstrap-colorpicker
+Site: http://mjolnic.com/bootstrap-colorpicker/
+License: Apache Software License, Version 2
 
-json2js
-=======
-http://www.JSON.org/json2.js
+Library: json2js
+Site: http://www.JSON.org/json2.js
 License: Public Domain
 
+Library: jquery-1.7.2
```

Comment:
We can remove jquery since I currently use the webgateway version (or link to cdn for the demo build)